src/web/app/pages-instructor/instructor-session-result-page/instructor-session-result-page.component.ts

```diff
@@ -383,7 +402,13 @@ export class InstructorSessionResultPageComponent extends InstructorCommentsComp
       ;
 
       response.subscribe(() => {
-        this.router.navigateByUrl('/web/instructor/sessions');
+        this.loadFeedbackSessionVisibleFromTime(this.courseId, this.fsName);
```

Comment:
I might be wrong, but I don't think you need this. Both `(unpublish|publish)FeedbackSession` endpoint should return the session object itself, which you can use straightaway to construct the new visible from time.

Reply:
Yes you are right. Thanks for pointing out. I should have checked the response type from the endpoint. I think part of the reason for me to overlook this is because the type of the variable `response` is `Observable<any>` where it actually can be `Observable<FeedbackSession>`.